### PR TITLE
Fix and add links in Computer Science roadmap

### PR DESCRIPTION
**Pull request by whoismtrx**
I made some changes to the Computer Science Roadmap. In the C# section, I edited the feed link because it was redirecting to the C feed. Additionally, I added some useful resources to the data structures and array sections.

### CODE REVIEW

File: src/data/roadmaps/computer-science/content/102-data-structures/index.md [hunk 1]
```diff
@@ -6,5 +6,7 @@ Visit the following resources to learn more:
 
 - [@article@What are Data Structures?](https://www.geeksforgeeks.org/data-structures)
 - [@article@ Data Structures and Algorithms](https://www.javatpoint.com/data-structure-tutorial)
+- [@course@ Data Structures and Algorithms By Google](https://techdevguide.withgoogle.com/paths/data-structures-and-algorithms/)
 - [@video@Data Structures Illustrated](https://www.youtube.com/watch?v=9rhT3P1MDHk\&list=PLkZYeFmDuaN2-KUIv-mvbjfKszIGJ4FaY)
+- [@video@A Great Data Structures playlist](https://youtube.com/playlist?list=PLDV1Zeh2NRsB6SWUrDFW2RmDotAfPbeHu&si=_EEf7x58G6lUcMGG)
```
Comment (dansholds) [inline src/data/roadmaps/computer-science/content/102-data-structures/index.md:11]:
```suggestion
- [@video@Data Structures playlist](https://youtube.com/playlist?list=PLDV1Zeh2NRsB6SWUrDFW2RmDotAfPbeHu&si=_EEf7x58G6lUcMGG)
```

**Comment by dansholds** [comment]
LGTM, next time please squash your commits before making the PR so it's clean 👍

**Comment by whoismtrx** [comment]
Noted, thanks mate!!

**Comment by dansholds** [comment]
@whoismtrx I went to give you the Contributor role on Discord...but you've left 🥹

**Comment by whoismtrx** [comment]
I think there's a problem with the discord invitation link, i tried to enter but nothing happened. Also i don't remember leaving the server.

**Pull request merged**